nb-adapters/adapter-qdrant/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>adapter-qdrant</artifactId>
+    <packaging>jar</packaging>
+
+    <parent>
+        <artifactId>mvn-defaults</artifactId>
+        <groupId>io.nosqlbench</groupId>
+        <version>${revision}</version>
+        <relativePath>../../mvn-defaults</relativePath>
+    </parent>
+
+    <name>${project.artifactId}</name>
+    <description>
+        An nosqlbench adapter driver module for the Qdrant database.
+    </description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.nosqlbench</groupId>
+            <artifactId>nb-annotations</artifactId>
+            <version>${revision}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.nosqlbench</groupId>
+            <artifactId>adapters-api</artifactId>
+            <version>${revision}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-protobuf</artifactId>
+            <!-- <version>1.63.0</version> -->
+            <!-- Trying to match https://github.com/qdrant/java-client/blob/v1.9.0/build.gradle#L80 -->
+            <version>1.59.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java-util</artifactId>
+            <!--<version>3.25.3</version>-->
+            <!-- Trying to match https://github.com/qdrant/java-client/blob/master/build.gradle#L81 -->
+            <version>3.24.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
This dependency will make it so that we can't enable this adapter by default. Guava is not well-behaved across versions where there are version conflicts in the same artifact.

Reply:
Yes, this driver is not enabled by default. Taken care https://github.com/nosqlbench/nosqlbench/pull/1942/files#diff-1014dc69475c4c9e5f16e809bae50284fdac951a044cb289fc04bd8415d0f51eR245 and https://github.com/nosqlbench/nosqlbench/pull/1942/files#diff-c4049021bae965d533aa8d45bc1d39e332e1e07a75a866c40214f089010fc1e7R180